core/arch/arm/tee/entry_std.c

```diff
@@ -240,7 +239,7 @@ static void entry_open_session(struct thread_smc_args *smc_args,
 
 out:
 	if (s)
-		arg->session = tee_svc_kaddr_to_uref(s);
+		arg->session = (vaddr_t)s;
```

Comment:
Yes, that's a problem before this P-R also.
This is not a real problem in any of our official platforms as the top 32-bits always are zero so far.